torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -8,6 +9,7 @@
 #include <stdexcept>
 #include <tuple>
 #include <unordered_set>
+#include <mutex>
```

Comment:
nit: remove?

Reply:
`std::call_once` is defined in `<mutex>`

Reply:
Sorry for not adding more context here. mutex is already included in ProcessGroupNCCL.h, so we may want to remove this line.

Reply:
fixed